[2025-04-03T04:39:40.576+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-03T04:39:40.651+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schedule.move_cloud_clientes scheduled__2025-03-23T08:00:00+00:00 [queued]>
[2025-04-03T04:39:40.698+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schedule.move_cloud_clientes scheduled__2025-03-23T08:00:00+00:00 [queued]>
[2025-04-03T04:39:40.703+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-03T04:39:41.263+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): move_cloud_clientes> on 2025-03-23 08:00:00+00:00
[2025-04-03T04:39:41.290+0000] {standard_task_runner.py:72} INFO - Started process 398 to run task
[2025-04-03T04:39:41.339+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'schedule', 'move_cloud_clientes', 'scheduled__2025-03-23T08:00:00+00:00', '--job-id', '1953', '--raw', '--subdir', 'DAGS_FOLDER/dag_master.py', '--cfg-path', '/tmp/tmps8_pbbth']
[2025-04-03T04:39:41.365+0000] {standard_task_runner.py:105} INFO - Job 1953: Subtask move_cloud_clientes
[2025-04-03T04:39:41.563+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-03T04:39:41.854+0000] {task_command.py:467} INFO - Running <TaskInstance: schedule.move_cloud_clientes scheduled__2025-03-23T08:00:00+00:00 [running]> on host dca2c4cc2d1b
[2025-04-03T04:39:41.866+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-03T04:39:42.491+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='schedule' AIRFLOW_CTX_TASK_ID='move_cloud_clientes' AIRFLOW_CTX_EXECUTION_DATE='2025-03-23T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-23T08:00:00+00:00'
[2025-04-03T04:39:42.524+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-03T04:39:42.696+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-03T04:39:42.699+0000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'docker exec linux_python /bin/bash -c \'/workspace/scripts/run_move_file.sh /path/to/move_cloud/clientes/clientes.csv table_ingestion_files/clientes/20250403/clientes_1743655180.csv && echo "Script Executado com Sucesso!"\'']
[2025-04-03T04:39:42.764+0000] {subprocess.py:99} INFO - Output:
[2025-04-03T04:39:44.407+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,387 - INFO - Iniciando o upload do arquivo '/path/to/move_cloud/clientes/clientes.csv' para o blob 'table_ingestion_files/clientes/20250403/clientes_1743655180.csv' no container 'filedataimport'.
[2025-04-03T04:39:44.450+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,390 - INFO - Blob URL gerada: https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/clientes/20250403/clientes_1743655180.csv?sp=cw&st=2024-10-20T02:42:12Z&se=2026-10-20T10:42:12Z&spr=https&sv=2022-11-02&sr=c&sig=MaoRwZWwtKEx5vDafwkbeHkx1Rir6bWRYy0FB3OOs58%3D
[2025-04-03T04:39:44.452+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,419 - INFO - Enviando o arquivo '/path/to/move_cloud/clientes/clientes.csv' para o Azure Blob Storage.
[2025-04-03T04:39:44.467+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:44,459 - INFO - Request URL: 'https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/clientes/20250403/clientes_1743655180.csv?sp=REDACTED&st=REDACTED&se=REDACTED&spr=REDACTED&sv=REDACTED&sr=REDACTED&sig=REDACTED'
[2025-04-03T04:39:44.470+0000] {subprocess.py:106} INFO - Request method: 'PUT'
[2025-04-03T04:39:44.480+0000] {subprocess.py:106} INFO - Request headers:
[2025-04-03T04:39:44.481+0000] {subprocess.py:106} INFO -     'Content-Length': '558955'
[2025-04-03T04:39:44.483+0000] {subprocess.py:106} INFO -     'x-ms-blob-type': 'REDACTED'
[2025-04-03T04:39:44.484+0000] {subprocess.py:106} INFO -     'If-None-Match': '*'
[2025-04-03T04:39:44.484+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-04-03T04:39:44.485+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/octet-stream'
[2025-04-03T04:39:44.486+0000] {subprocess.py:106} INFO -     'Accept': 'application/xml'
[2025-04-03T04:39:44.487+0000] {subprocess.py:106} INFO -     'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.12.9 (Linux-6.10.14-linuxkit-aarch64-with-glibc2.36)'
[2025-04-03T04:39:44.489+0000] {subprocess.py:106} INFO -     'x-ms-date': 'REDACTED'
[2025-04-03T04:39:44.491+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': 'aa4c344c-1045-11f0-a9d3-0242ac120004'
[2025-04-03T04:39:44.491+0000] {subprocess.py:106} INFO - A body is sent with the request
[2025-04-03T04:39:45.311+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:45,285 - INFO - Response status: 409
[2025-04-03T04:39:45.355+0000] {subprocess.py:106} INFO - Response headers:
[2025-04-03T04:39:45.363+0000] {subprocess.py:106} INFO -     'Content-Length': '220'
[2025-04-03T04:39:45.367+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/xml'
[2025-04-03T04:39:45.367+0000] {subprocess.py:106} INFO -     'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
[2025-04-03T04:39:45.369+0000] {subprocess.py:106} INFO -     'x-ms-request-id': 'a9fa2f97-701e-0015-5752-a4f93d000000'
[2025-04-03T04:39:45.380+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': 'aa4c344c-1045-11f0-a9d3-0242ac120004'
[2025-04-03T04:39:45.399+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-04-03T04:39:45.400+0000] {subprocess.py:106} INFO -     'x-ms-error-code': 'BlobAlreadyExists'
[2025-04-03T04:39:45.402+0000] {subprocess.py:106} INFO -     'Date': 'Thu, 03 Apr 2025 04:39:44 GMT'
[2025-04-03T04:39:45.403+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:45,297 - ERROR - Erro ao enviar o arquivo '/path/to/move_cloud/clientes/clientes.csv': The specified blob already exists.
[2025-04-03T04:39:45.404+0000] {subprocess.py:106} INFO - RequestId:a9fa2f97-701e-0015-5752-a4f93d000000
[2025-04-03T04:39:45.404+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:45.1889540Z
[2025-04-03T04:39:45.405+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-04-03T04:39:45.405+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-04-03T04:39:45.406+0000] {subprocess.py:106} INFO - RequestId:a9fa2f97-701e-0015-5752-a4f93d000000
[2025-04-03T04:39:45.406+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:45.1889540Z</Message></Error>
[2025-04-03T04:39:45.409+0000] {subprocess.py:106} INFO - 2025-04-03 04:39:45,298 - ERROR - Falha no processo de upload: The specified blob already exists.
[2025-04-03T04:39:45.415+0000] {subprocess.py:106} INFO - RequestId:a9fa2f97-701e-0015-5752-a4f93d000000
[2025-04-03T04:39:45.418+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:45.1889540Z
[2025-04-03T04:39:45.419+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-04-03T04:39:45.425+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-04-03T04:39:45.426+0000] {subprocess.py:106} INFO - RequestId:a9fa2f97-701e-0015-5752-a4f93d000000
[2025-04-03T04:39:45.431+0000] {subprocess.py:106} INFO - Time:2025-04-03T04:39:45.1889540Z</Message></Error>
[2025-04-03T04:39:45.550+0000] {subprocess.py:106} INFO - Script Executado com Sucesso!
[2025-04-03T04:39:45.585+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-03T04:39:45.697+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-03T04:39:45.702+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=schedule, task_id=move_cloud_clientes, run_id=scheduled__2025-03-23T08:00:00+00:00, execution_date=20250323T080000, start_date=20250403T043940, end_date=20250403T043945
[2025-04-03T04:39:45.930+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-03T04:39:45.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
